integration_tests/macros/test_snakeify_case.sql

```diff
@@ -0,0 +1,56 @@
+{# This tests the output of a dummy set of inputs to the snakeify case macro to ensure that it returns what we expect to come out does.
+This doesn't run on any actual data, we are just comparing the text that is generated.
+
+It runs 9 tests:
+1) a single world, no change
+2) already snake case, no change
+3) pascal case
+4) camel case, basic
+5) camel case with multiple workds
+6) camel case ending with a series of capitals
+7) camel case with a series of capitals in the middle
+8) camel case with hypthons
```

Comment:
Should be *hyphen I think?

Reply:
I looked at this word and said to myself it didn't look right, but then just went with it anyway. Too much thinking about python...